go.mod

```diff
@@ -4,7 +4,7 @@ go 1.23.2
 
 require (
 	github.com/conduitio/conduit-commons v0.5.0
-	github.com/conduitio/conduit-connector-sdk v0.12.0
+	github.com/conduitio/conduit-connector-sdk v0.12.1-0.20241128142715-415d5f0909e0
```

Comment:
Reminder to update this dependency when https://github.com/ConduitIO/conduit-connector-sdk/pull/211 is merged.